packages/nextjs/scaffold.config.ts

```diff
@@ -37,6 +42,20 @@ const scaffoldConfig = {
   // .env.local for local testing, and in the Vercel/system env config for live apps.
   alchemyApiKey: process.env.NEXT_PUBLIC_ALCHEMY_API_KEY || "oKxs-03sij-U_N0iOlrSsZFr29-IqbuF",
 
+  // Burner Wallet configuration
+  burnerWallet: {
+    // Set it to false to completely remove burner wallet from all networks
+    enabled: true,
+    // Only show the Burner Wallet when running on hardhat network
+    onlyLocal: false,
```

Comment:
I think this should default to "true"? What do you think @technophile-04 ?

Reply:
Umm actually se-1 has it by default but yeah I feel since se-2 is bit more on side of production  build it makes sense to make it true, updated it at [3a96074](https://github.com/scaffold-eth/se-2/pull/257/commits/3a96074eaa9ad9c11bba0ed033a7c6dbe1aaf8be) we should probably mention it in the README.md

Reply:
Just wondering should I update the README.md in this PR itself ? I feel we should have a small dedicated section to `scaffold.config` in README.md a minimal section